File: CodeCademy_8Ball.js
let userName = ''

userName = true ? `Hello, ${userName}, what is your question?` : 'Hello, Stranger, what is your question?'
console.log(userName)

userQuestion = `My question is: `
console.log(userQuestion)

randomNumber = Math.floor(Math.random() * 8)
eightBall = ''

switch (randomNumber){
  case (0):
    console.log('It is certain');
    break;
  case (1):
    console.log('It is decidedly so');
    break;
  case (2):
    console.log('Reply hazy, try again');
    break;
  case (3):
    console.log('Cannot predict now');
    break;
  case (4):
    console.log('Do not count on it');
    break;
  case (5):
    console.log('My sources say no');
    break;
  case (6):
    console.log('Outlook not so good');
    break;
  case (7):
    console.log('Signs point to yes')
    break;
  default:
    console.log('Try again')}

File: CodeCademy_8Ball.py
name = ""
question = "Will I find a new job?"
answer = ""

import random

random_number = random.randint(1, 9)

print(random_number)

if random_number == 1:
  answer = "Yes-definetly"
elif random_number == 2:
  answer = "It is decidedly so"
elif random_number == 3:
  answer = "Without a doubt"
elif random_number == 4:
  answer = "Reply hazy, try again"
elif random_number == 5:
  answer = "Ask again later"
elif random_number == 6:
  answer = "Better not tell you now"
elif random_number == 7:
  answer = "My sources say no"
elif random_number == 8:
  answer = "Outlook not so good"
elif random_number == 9:
  answer = "Doubtful"
else:
  answer = "Error"

if name == "":
  print("Question: " + question)
else:
  print(name + " : " + question)

if question == "":
  print("Do you have a question?")
else:
  print("Magic 8-ball's answer: " + answer)

